src/Maker/MakeAuthenticator.php

```diff
@@ -233,13 +254,15 @@ private function generateAuthenticatorClass(array $securityData, string $authent
     {
         // generate authenticator class
         if (self::AUTH_TYPE_EMPTY_AUTHENTICATOR === $authenticatorType) {
-            $this->generator->generateClass(
-                $authenticatorClass,
-                'authenticator/EmptyAuthenticator.tpl.php',
-                [
+            $template = 'authenticator/EmptyAuthenticator.tpl.php';
```

Comment:
```suggestion
            $template = sprintf('authenticator/%sEmptyAuthenticator.tpl.php', $this-> useSecurity52 ? 'Security52' : '');
```

then no if below

Reply:
Hmm, in fact, I'd inline this into the `generateClass` 2nd argument.

Reply:
done